.markdownlint.json

```diff
@@ -1,8 +1,11 @@
 {
+  "$schema": "https://raw.githubusercontent.com/DavidAnson/markdownlint/main/schema/markdownlint-config-schema.json",
   "no-inline-html": false,
   "line-length": false,
   "no-trailing-punctuation": false,
-  "no-duplicate-header": false,
+  "no-duplicate-heading": {
+    "siblings_only": true
```

Comment:
Why did we change this config?

Reply:
FWIW I'm in favor of enabling `siblings_only`. https://github.com/DavidAnson/markdownlint/issues/1601